### PR TITLE
[python 3.12 support] Drop python 3.8, upgrade pylint-django

### PR DESCRIPTION
**Pull request by Pierre-Sassoulas**
This should be the last blocker for python 3.12 / pylint 3.0 support.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13-dev"]
+        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
```
Comment (valeriupredoi) [inline .github/workflows/tests.yml:16]: have you checked compatibility with the new 3.13?

File: pyproject.toml [hunk 2]
```diff
@@ -18,7 +18,6 @@ classifiers = [
     "Intended Audience :: Developers",
     "Operating System :: Unix",
     "Topic :: Software Development :: Quality Assurance",
-    "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
```
Comment (valeriupredoi) [inline pyproject.toml:23]: add 3.13 here if it is compatible/supported?

**Review by valeriupredoi — APPROVED** [review]
cheers @Pierre-Sassoulas - looking good, just two quick questions from me :beer:

File: .github/workflows/tests.yml (hunk 1 above)
Comment (Pierre-Sassoulas) [reply .github/workflows/tests.yml:16]: The latest pylint-django is not compatible with python 3.13, but the CI's tests are not catching that (2 tests are failing in the pylint-django pipeline, which doesn't mean that prospector is incompatible, there's just an issue in pylint-django).

File: pyproject.toml (hunk 2 above)
Comment (Pierre-Sassoulas) [reply pyproject.toml:23]: It was already there (2 lines below).

File: .github/workflows/tests.yml (hunk 1 above)
Comment (Pierre-Sassoulas) [reply .github/workflows/tests.yml:16]: We already had python 3.13 tests passing (running on 3.13rc2 as far as I know)

File: pyproject.toml (hunk 2 above)
Comment (valeriupredoi) [reply pyproject.toml:23]: ah my bad, cheers!

File: .github/workflows/tests.yml (hunk 1 above)
Comment (valeriupredoi) [reply .github/workflows/tests.yml:16]: gotcha, thanks! :beer:

**Review by sbrunner — APPROVED** [review]

**Comment by Pierre-Sassoulas** [comment]
Anyone want to release 1.12.1 ? :)

**Comment by valeriupredoi** [comment]
Great work! I'll take care of the conda package then, later today 👍🍺

**Comment by valeriupredoi** [comment]
OK folks, 1.2.1 happily out on conda-forge via https://github.com/conda-forge/prospector-feedstock/pull/50 :beer:

**Pull request merged**